pom.xml

```diff
@@ -8,7 +8,7 @@
     <parent>
         <groupId>org.carlspring.strongbox</groupId>
         <artifactId>strongbox-parent</artifactId>
-        <version>1.0-SNAPSHOT</version>
+     <version>1.0-Issue1876-SNAPSHOT</version>
```

Comment:
@MrunalRajput : Please, preserve the original formatting. :)

@steve-todorov: Will this version + branch format be accepted by Jenkins?

Reply:
The versioning is incorrect - should be:
```suggestion
     <version>1.0-issue-1876-SNAPSHOT</version>
```

Otherwise won't deploy and tests down the pipeline will fail :)

Reply:
Thanks! I will fix this.

Reply:
I will re open a new PR with correct branch name.